media-gfx/eom/eom-1.28.0-r1.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit mate optfeature
+
+MINOR=$(($(ver_cut 2) % 2))
+if [[ ${MINOR} -eq 0 ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
+fi
+
+DESCRIPTION="The MATE image viewer"
+LICENSE="FDL-1.1+ GPL-2+ IJG LGPL-2+"
+SLOT="0"
+
+IUSE="X debug exif imagemagick +introspection nls jpeg lcms svg tiff xmp"
+
+COMMON_DEPEND="
+	>=app-accessibility/at-spi2-core-2.46.0
+	>=dev-libs/glib-2.52:2
+	>=dev-libs/libpeas-1.8.0:0[gtk]
+	>=dev-libs/libxml2-2:2
+	gnome-base/dconf
+	>=mate-base/mate-desktop-1.17.0
+	sys-libs/zlib
+	x11-libs/cairo
+	>=x11-libs/gdk-pixbuf-2.36.5:2[introspection?,jpeg?,tiff?]
+	>=x11-libs/gtk+-3.22:3[introspection?]
+	x11-libs/libX11
+	>=x11-misc/shared-mime-info-0.20
+	exif? (
+		>=media-libs/libexif-0.6.22
+		media-libs/libjpeg-turbo:=
+	)
+	imagemagick? ( >=media-gfx/imagemagick-6.2.6 )
+	introspection? ( >=dev-libs/gobject-introspection-0.9.3:= )
+	jpeg? ( media-libs/libjpeg-turbo:= )
+	lcms? ( media-libs/lcms:2 )
+	svg? ( >=gnome-base/librsvg-2.36.2:2 )
+	xmp? ( >=media-libs/exempi-1.99.5:2= )
+"
+
+RDEPEND="${COMMON_DEPEND}
+	virtual/libintl
+"
+
+BDEPEND="${COMMON_DEPEND}
+	app-text/yelp-tools
+	dev-util/glib-utils
+	dev-util/gtk-doc
+	dev-build/gtk-doc-am
+	>=sys-devel/gettext-0.19.8
+	virtual/pkgconfig
+"
+
+src_configure() {
+	mate_src_configure \
+		--enable-thumbnailer \
+		$(use_enable debug) \
+		$(use_enable introspection) \
+		$(use_with X x) \
+		$(use_with exif libexif) \
+		$(usex imagemagick \
+			--without-gdk-pixbuf-thumbnailer \
+			--with-gdk-pixbuf-thumbnailer \
+		) \
+		$(use_with jpeg libjpeg) \
+		$(use_with lcms cms) \
+		$(use_with svg librsvg) \
+		$(use_with xmp)
+}
+
+pkg_postinst() {
+	optfeature "For viewing webp images please install" gui-libs/gdk-pixbuf-loader-webp
+}
```

Comment:
This is going to print:

```
 * Install additional packages for optional runtime features:
 *   gui-libs/gdk-pixbuf-loader-webp for For viewing webp images please install
```

Reply:
@oz123 for future reference you might want to look at the optifeature_header option as show in https://devmanual.gentoo.org/eclass-reference/optfeature.eclass/index.html and https://github.com/gentoo/gentoo/blob/bd9a188b2ca57a74a27ce74686ac1d46b01f2c35/sys-fs/fstransform/fstransform-0.9.4.ebuild#L18

However I'll be happier just "To enable webp support" in this case please.

Reply:
Thanks for reviewing this. I was quite puzzled why the optfeature message was not showing, so it was hard to verify the output.
However, the source code of the eclass is very explicit:

> Print out a message suggesting an optional package (or packages)
> not currently installed which provides the described functionality.

So now I was able to verify that the message is:
```
>>> Installing (1 of 1) media-gfx/eom-1.28.0-r1::oz123-overlay
 * Updating .desktop files database ...                                                                                                                                                                                               [ ok ]
 * Updating icons cache ...                                                                                                                                                                                                           [ ok ]
 * Updating GSettings schemas ...                                                                                                                                                                                                     [ ok ]
 * Install additional packages for optional runtime features:
 *   gui-libs/gdk-pixbuf-loader-webp for webp support
```